docker/elo_db/db_schema.sql

```diff
@@ -105,6 +105,10 @@ ALTER TABLE `games`
   ADD CONSTRAINT `games_ibfk_2` FOREIGN KEY (`looser_user_nid`) REFERENCES `users` (`user_nid`) ON DELETE CASCADE;
 COMMIT;
 
-/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
-/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
-/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
+CREATE TABLE `password` (
+  `password` varchar(40) NOT NULL,
+  PRIMARY KEY (`password`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO `password` (`password`) VALUES
+('cd798c453817754e7fab67c9572fc9d3182a2ee2');
```

Comment:
To jest zaszyfrowane: 'elo'. Napisałem o tym w readme.ME. Jak ktoś sobie postawi apke to żeby odrazu mu działało.